Samples/Samples/View/CalendarEventAddPage.xaml.cs

```diff
@@ -11,5 +11,37 @@ public CalendarEventAddPage()
         {
             InitializeComponent();
         }
+
+        int maxIntervalLength = 2;
+
+        void Entry_OnTextChanged(object sender, TextChangedEventArgs e)
+        {
```

Comment:
What rule are we trying to enforce here? This type of functionality should be in the view model

Reply:
Shouldn't really be required, realised I could just use MaxLength="2" combined with Keyboard="Number" to prevent more than 2 digits being entered